race/disabled.go

```diff
@@ -0,0 +1,9 @@
+//go:build !race
+
+package race
+
+// We can use this constant to determine whether or not the Go Race Detector is enabled. If the race detector is enabled,
+// the `race` build tag is passed, so this file won't be compiled, where the other one (./enabled.go) will be.
+// That is, if the race detector is enabled, the constant below will be false. Otherwise, it'll be true.
+// It's kinda weird, but it's also how go itself does it: https://github.com/golang/go/blob/master/src/internal/race/race.go#L15
```

Comment:
Huh, that's an interesting way to do it... But it's not dumb if it works.

Reply:
i feel like "prior art from the creators of the language" is about as carte blanche as it gets to do janky shit, and yet... it still feels incredibly janky.